public/templates/tracks/stats.html

```diff
@@ -0,0 +1,10 @@
+<div class="chart-btn-container">
+    <a href="{{::track.url}}" class="btn chart-exit">Return to track</a> <button class="btn btn-primary" ng-class="{ selected: totalSelected }" ng-click="totalClick()">Cumulative</button> <button class="btn btn-primary" ng-class="{ selected: !totalSelected }" ng-click="dailyClick()">{{ dailyText }}</button>
+</div>
+<h1>Geeky Stats For Geeky People (BETA)</h1>
```

Comment:
Once it launches into production and is fully tested. I also plan to add some more stats and improve the layout